ios/CodePush/CodePushPackage.m

```diff
@@ -292,26 +292,37 @@ + (NSString *)getCodePushPath
 
 + (NSDictionary *)getCurrentPackage:(NSError **)error
 {
-    NSString *packageHash = [CodePushPackage getCurrentPackageHash:error];
-    if (*error || !packageHash) {
+    NSError *localError;
+    NSString *packageHash = [CodePushPackage getCurrentPackageHash:&localError];
+    if (!packageHash) {
+        if (localError && error) {
+            *error = localError;
+        }
```

Comment:
I think if we passed `error` directly instead of `&localError`, we could simply have

```
if (!packageHash) {
    return nil;
}
```

and get rid of these lines:
```
if (localError && error) {
    *error = localError;
}
```